po/ar.po

```diff
@@ -347,129 +347,129 @@ msgstr "تمرين: فيبوناتشي"
 msgid "Standard Library"
-msgstr ""
+msgstr "المكتبة القياسية"
 
 #: src/SUMMARY.md src/std-types/docs.md
 msgid "Documentation"
-msgstr ""
+msgstr "المستندات التوثيقية (Documentation)"
 
 #: src/SUMMARY.md
 msgid "`Option`"
-msgstr ""
+msgstr "الخيار (Option)"
 
 #: src/SUMMARY.md src/error-handling/result.md
 msgid "`Result`"
-msgstr ""
+msgstr "النتيجة (Result)"
 
 #: src/SUMMARY.md src/android/aidl/types/primitives.md
 #: src/android/interoperability/cpp/type-mapping.md
 #, fuzzy
 msgid "`String`"
-msgstr "السلاسل النص او الكتابة (Strings)"
+msgstr "السلاسل النصية (Strings)"
 
 #: src/SUMMARY.md src/std-types/vec.md
 msgid "`Vec`"
-msgstr ""
+msgstr "المصفوفات (Vectors)"
 
 #: src/SUMMARY.md src/std-types/hashmap.md src/bare-metal/no_std.md
 msgid "`HashMap`"
-msgstr ""
+msgstr "خريطة المشفرة (HashMap)"
 
 #: src/SUMMARY.md src/std-types/exercise.md
 #, fuzzy
 msgid "Exercise: Counter"
-msgstr "تمرين: فيبوناتشي"
+msgstr "تمرين: عداد"
 
 #: src/SUMMARY.md src/std-traits.md
 msgid "Standard Library Traits"
-msgstr ""
+msgstr "سمات المكتبة القياسية"
 
 #: src/SUMMARY.md src/std-traits/comparisons.md
 #: src/concurrency/welcome-async.md
 msgid "Comparisons"
-msgstr ""
+msgstr "مقارنات"
 
 #: src/SUMMARY.md src/std-traits/operators.md
 msgid "Operators"
-msgstr ""
+msgstr "المشغلات (`Operators`)"
 
 #: src/SUMMARY.md src/std-traits/from-and-into.md
 msgid "`From` and `Into`"
-msgstr ""
+msgstr "السمات (من و إلى) (From and Into)"
 
 #: src/SUMMARY.md src/std-traits/casting.md
 msgid "Casting"
-msgstr ""
+msgstr "التحويل"
 
 #: src/SUMMARY.md src/std-traits/read-and-write.md
 msgid "`Read` and `Write`"
-msgstr ""
+msgstr "السمات (القراءة و الكتابة) (Read and Write)"
 
 #: src/SUMMARY.md
 msgid "`Default`, struct update syntax"
-msgstr ""
+msgstr "الإفتراضي، بناء جملة تحديث البنية (Default, struct update syntax)"
 
 #: src/SUMMARY.md src/std-traits/closures.md
 msgid "Closures"
-msgstr ""
+msgstr "الإغلاقات"
 
 #: src/SUMMARY.md src/std-traits/exercise.md
 msgid "Exercise: ROT13"
-msgstr ""
+msgstr "تمرين: ROT13"
 
 #: src/SUMMARY.md
 #, fuzzy
 msgid "Day 3: Morning"
-msgstr "اليوم الأول: صباحا"
+msgstr "اليوم الثالث: صباحا"
 
 #: src/SUMMARY.md src/memory-management.md
 msgid "Memory Management"
-msgstr ""
+msgstr "إدارة الذاكرة"
 
 #: src/SUMMARY.md src/memory-management/review.md
 msgid "Review of Program Memory"
-msgstr ""
+msgstr "مراجعة ذاكرة البرنامج"
 
 #: src/SUMMARY.md src/memory-management/approaches.md
 msgid "Approaches to Memory Management"
-msgstr ""
+msgstr "طرق إدارة الذاكرة"
 
 #: src/SUMMARY.md src/memory-management/ownership.md
 msgid "Ownership"
-msgstr ""
+msgstr "الملكية"
 
 #: src/SUMMARY.md src/memory-management/move.md
 msgid "Move Semantics"
-msgstr ""
+msgstr "دلالات النقل"
 
 #: src/SUMMARY.md
 msgid "`Clone`"
-msgstr ""
+msgstr "الاستنساخ"
 
 #: src/SUMMARY.md src/memory-management/copy-types.md
 msgid "Copy Types"
-msgstr ""
+msgstr "أنواع النسخ"
 
 #: src/SUMMARY.md
 msgid "`Drop`"
-msgstr ""
+msgstr "الإسقاط"
```

Comment:
Another bit of code:

```suggestion
msgstr "`Drop`"
```

If you like, you could consider writing the Arabic translation after the English work, most likely in parenthesis if that makes sense for Arabic. We don't have strict guidelines for this, all we hope for is that the translators are consistent within a given language.

See [the style guide](https://github.com/google/comprehensive-rust/blob/main/STYLE.md#translations) for a bit of advise here. Feel free to suggest more tips and advice as you develop it!

Reply:
Ah, I see you added the English words in parenthesis above. That seems nice to me.